src/LanguageServer/Test/GoToDefinitionTests.cs

```diff
@@ -165,6 +165,45 @@ class D(C):
             reference.range.Should().Be(2, 4, 2, 5);
         }
 
+        [TestMethod, Priority(0)]
+        public async Task GotoDefinitionFromParentOtherModuleOnSuper() {
+            var otherModPath = TestData.GetTestSpecificUri("other.py");
+            var testModPath = TestData.GetTestSpecificUri("test.py");
+            const string otherModCode = @"
+class C:
+    v: int
+    def test(self):
+        pass
+";
+            const string testModCode = @"
+from other import C
+
+class D(C):
+    def hello(self):
+        super().test();
+        super().v
```

Comment:
Issue with goto definition on data member super().v

This test is copied from the test above ```GotoDefinitionFromParentOtherModule()```

```DefinitionSources::FromMemberExpression(..)``` seems to special case data members but only if the type is ```IPythonClassType``` , currently my ```PythonSuperType``` is not implementing that interface so then it falls back to just calling ``` if (type?.GetMember(mex.Name) is ILocatedMember lm) { ``` but i guess data members dont track their declaration location.

Reply:
I would add `interface IMultipleLocationsMember: IEnumerable<Location>` which would be able to store multiple locations - i.e. all bases. In DefinitionSource code then can check if item is `IMultipleLocationsMember` and fetch and return all locations so in case of multiple inheritance user would see list with choices.

Reply:
I'm not sure if a multi-location is needed. If you search the MRO for the correct member, then that's going to have a location and that's where you'd go, like any other access. super is only going to give back the first found member for any given name, and I think the only way to get a specific base's member would be to pass more arguments to super to choose an alternate MRO, or refer to the type directly and bind its functions.

Finding alternative things that could be called based on children classes (i.e. goto impl) is another story, though.

Reply:
OK